GOVERNANCE.md

```diff
@@ -0,0 +1,356 @@
+
+TLC members fulfill all requirements of Committers, and also:
+
+* May merge external pull requests for accepted issues upon reviewing and approving
+the changes.
+* Push code directly to the repos or, when necessary, create and merge their own pull
+requests once they have collected the feedback they deem necessary.
+* Discuss once a month the technical status and project roadmap.
+
+To become a TLC member:
+
+* Work in a helpful and collaborative way with the community.
+* Have given good feedback on others' submissions and displayed an overall understanding
+of the code quality standards for the project.
+* Commit to being a part of the community for the long-term.
+* Have submitted a minimum of 20 qualifying pull requests or pushed a minimum of 20 qualifying commits across the different projects (IDE, Docker-Selenium, Selenium, Site & Docs).
+
+A Committer is invited to become a TLC member by existing TLC members. A nomination
+will result in discussion and then a decision by the TLC.
+
+#### Process for Adding TLC Members
+
+1. Add the GitHub user to the `Selenium TLC` team
+1. Set the GitHub user to have the "Owner" role for the SeleniumHQ organization
+1. Invite to the Slack TLC chat room (`selenium-tlc`)
+1. Add the TLC member to the different package distribution organizations
+    * NPM
+    * SonarType (maven)
+    * pypi.org
+    * rubygems.org
+    * nuget.org
+1. Tweet congratulations to the new TLC member from the SeleniumHQ Twitter account
+
+The current TLC members are:
+
+* [@barancev](https://github.com/barancev)
+* [@jimevans](https://github.com/jimevans)
+* [@shs96c](https://github.com/shs96c)
+* [@titusfortner](https://github.com/titusfortner)
+
+### Project Leadership Committee (PLC)
+
+The overall continuity and future of the project is overseen by the Project
+Leadership Committee (PLC), which acts as a bridge between the Selenium project
+and the Software Freedom Conservancy (SFC).
+
+Since there are many different facets to the Selenium project, the PLC wants to
+reflect more than just people who sling code about. The PLC gets involved in
+different ways, such as whenever the project spends money, enters legal agreements,
+or has to deal with lawyers. Typically the PLC will discuss the topic with the rest
+of the community, and then a voting session will take place.
+
+PLC seats are not time-limited. Only one person may be affiliated with the any
+given employer can be on the PLC at a time. The PLC member count should be always
+odd so that no votes are ever tied, an ideal minimum number is 5. PLC members
+should also be active members of the community.
+
+To become a PLC member:
+
+* An existing PLC member steps down and suggests someone (or some people) to replace them.
+* Be a committer or a contributor.
+* Commit to being a part of the community for the long-term.
+* The existing PLC consults with the TLC and the active Committers, who vote on whether or
```

Comment:
as about wrt consensus-seeking vs voting

Reply:
I pushed a change with this, please have a look. Hopefully it looks better now.